quickstarts/dotnet/fubumvc/config.yml

```diff
@@ -1,7 +1,25 @@
-summary: Monitor FubuMVC with New Relic's .NET agent
+description: |
+  ## New Relic + FubuMVC
+  
+  New Relic’s monitoring quickstart for FubuMVC monitors the performance and reliability of FubuMVC applications. 
+  
+  It includes visual dashboards which showcase real-time metrics of the FubuMVC stack, including:
+  - Errors overview
+  - VM overview
+  - Latest error display
+  - CPU utilization display
+  - Amount of memory heap used, and more. 
+  The quickstart also includes alerts when certain metrics such as the Apdex score, memory usage, and number of transaction errors hit a critical threshold.
+  
+  ### Why monitor FubuMVC with New Relic?
+  
+  Building server-side systems can be a complex process. Between dealing with incoming and outgoing HTTP requests, interfacing with other APIs and backends, the nuances of the server-side code itself, and any sort of distributed interaction or job scheduling, there are many things that can go wrong. When you monitor your Fubumvc instance, it’s much easier to spot these errors before they occur. FubuMVC fails, your whole application will too. New Relic makes it easier to prevent that failure.
+  For example, if you get an  alert when your transaction errors hit a critical a threshold, it can give you advance notice about a dead server, a network bottleneck, or an attack against your web app. Similarly, the CPU utilization dashboard helps you to know when your server load is high and when traffic might be about to overwhelm a particular HTTP service. You can address your issues proactively, before they cause an outage.
+
+summary: |
+  FubuMVC is a framework for building server-side .NET applications. Monitoring FubuMVC is critical to ensuring that your application stays healthy. The New Relic quickstart provides key dashboards and alerts to oversee FubuMVC applications."## What is FubuMVC?\n\nA .Net framework for enabling teams to efficiently build server-side systems.\n\n## Get started!\n\nLeverage community expertise and instantly get value out of your telemetry data. This quickstart automatically instruments FubuMVC with the New Relic .Net agent, and allows you to instantly monitor your .Net application with out-of-the-box dashboards and alerts. Further leverage New Relic's APM capabilities by setting up [errors inbox](https://docs.newrelic.com/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/), [transaction tracing](https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/), and [service maps](https://docs.newrelic.com/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/).\n\n## More info\n\nCheck out the [documentation](https://docs.newrelic.com/docs/agents/net-agent/getting-started/introduction-new-relic-net/) to learn more about New Relic monitoring for FubuMVC. "
```

Comment:
This summary seems very long for a summary, the limit on summaries is 250 characters, so if you can shorten this that would be ideal.